### DISCUSSION TITLE
"Letterbox" ("Briefkasten" in German, "amenity=post_box" in OSM) within the "Things" overlay and other suggestions

### DISCUSSION THREAD

**Discussion by sushifan233** [streetcomplete/StreetComplete#5702] [Ideas]
First of all, many thanks for this great peace of software. Using this app, I started contributing to OSM and still I'm using it a lot. Now I want to make the following suggestions:

1. Please add the possibility to select "Letterbox" ("Briefkasten" in German, "amenity=post_box" in OSM) within the "Things" overlay (or did I miss something?)
2. It would be nice to be able to select not only one overlay but:
     - "Places" together with "Things"
     - "Addresses" togehter with "Buildings"
     - or all 4 overlays mentioned above
3. If you zoom out not all existing POIs (e.g. Things or Places) close together can be shown on the map. In this case it would be nice to have some small dots on the map similar to the dots for quests.
4. It would be nice to have red markings (e.g. "#" or a question mark) on buildings (overlay "Addresses") where the housenumber is missing. Currently it is rather difficult to find such houses.
5. On the other hand, the blue markings of buildings in the overlay "Buildings" is to much information, the type symbol already gives this info.
6. Last but not least I suggest to create a global setting "Show the appropriate OSM tag" for experienced users. If ON the app displays the OSM tag ("tagname=tag content") which is created or updated by answering a quest or creating a new POI.

Many thanks for your support

**Comment by nobkd** [comment]
1. is a duplicate of #4916 and #5699

**Comment by mnalis** [comment]
1. (already answered above, duplicate of https://github.com/streetcomplete/StreetComplete/issues/4916)
2. there are severe issues with idea of doing multiple overlay at once, see e.g. https://github.com/streetcomplete/StreetComplete/issues/5261#issuecomment-1732270014 and around 
3. Note that there is impending change to another rendering library MapLibre (https://github.com/streetcomplete/StreetComplete/issues/3123) so it is likely that rendering UI will change somewhat. After that has happened (watch the _"What's new"_ on app upgrades!), if the UI might benefit from improvements, it could be suggested (when opening GitHub issues, please fill the template, and always request **only one** idea in **one** Issue - that way they can be tracked and implemented, and potential issues with one won't block everything).
4. that sounds like a nice idea to me too; free free to open new Feature request issue [here](https://github.com/streetcomplete/StreetComplete/issues/new/choose). Just fill in the information that template asks of you!
5. building type icons are only shown at some zoom levels (zoom out more and they're gone), and colors provide much easier recognition allowing brain to instantly seen at a glance if everything is same color e.g. in residential quarter (as opposed to have to compare icons one by one which is much slower and prone to error)
6. there is already such button - after answering quest, press the "Undo" button in lower left corner and check the tags that were changed - it will ask you to confirm if you wish to undo it. It is recommended that you go to `Settings` and put `Upload answers automatically` to `Off` if you often intend to really undo the changes (e.g. if you just want to check how some answer would be tagged, but don't really want to answer the question). See also [SC FAQ: How can I learn exactly what tags StreetComplete changes?](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#How_can_I_learn_exactly_what_tags_StreetComplete_changes?)

**Reply by sushifan233** [reply]
Many thanks for your answers. I think I have to do much more searching and reading before posting ;-)
